.github/workflows/nodejs.yml

```diff
@@ -7,7 +7,7 @@ on:
   push:
     branches: [ develop, master, 2.0 ]
   pull_request:
-    branches: [ develop, 2.0 ]
+  workflow_dispatch:
```

Comment:
What does this workflow_dispatch do?

Reply:
It allows you to run a workflow on command in the Github Actions tab.

Reply:
Do we still need to specify branches on `pull_reqeust:`?

Reply:
Well we want tests to run on every PR, so I don't think we should. I left the branches on `push` so we don't have that issue where every PR has tests running twice (one for the push, one for the PR), which is true in the xrpl-py repo. But those 3 branches don't have PRs, so they should run when you push/merge a PR into them.